### DISCUSSION TITLE
MeshSideSetGenerator is gone?

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#21343] [Q&A Meshing] [answered]
MOOSE Team,

"MeshSideSetGenerator" seems to be gone from the updated MOOSE.
If so, please let me know an alternative app.

S. Thomas

**Comment by GiudGiud** [comment, marked as answer]
Hello

We removed it as it was redundant with the LowerDBlockFromSideSetGenerator
https://mooseframework.inl.gov/source/meshgenerators/LowerDBlockFromSidesetGenerator.html

Guillaume

**Reply by styyokuda** [reply]
Thank you Guillaume,

I am studying LowerDBlockFromSidesetGenerator.  It seems that elements generated by LowerDBlockFromSidesetGenerator do not have shape functions.
We have been using MeshSideSetGenerator to generate one-dimension-lower finite elements where we assign the diffusion equations with source terms.
We need MeshSideSetGenerator for our model we have been developing for years.
Could you please put MeshSideSetGenerator back into MOOSE?

S. Thomas

**Reply by GiudGiud** [reply]
Hello

What do you mean they do not have shape functions?
Looking at the code, the lowerDBlockFromSidesetGenerator does the same thing but it works for distributed mesh when the other one doesnt. 
It should work as is for your case. What errors are you getting?

Guillaume

**Reply by styyokuda** [reply]
Guillaume,

I got confused by the documentation.
I replaced MeshSideSetGenerator with LowerDBlockFromSidesetGenerator and it seems working. :)
It was too simple to believe.
I would like to thank you for your time and help.
Thomas
